src/components/Chat/Header.js

```diff
@@ -115,7 +116,7 @@ const Header = ({onBack}) => {
               focusChatName: true
             })
           }>
-          <IconEP name="dots-three-vertical" size={12.87} color={'#fff'} />
+          <IconEP name="dots-three-vertical" size={12.87} color={COLORS.white} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Replacing the hardcoded color value with a dynamic one from the theme object is a good practice. This makes it easier to manage and update colors across the application.
```diff
- <IconEP name="dots-three-vertical" size={12.87} color={'#fff'} />
+ <IconEP name="dots-three-vertical" size={12.87} color={COLORS.white} />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->